packages/sol-rayz/src/getParsedAccountByMint.ts

```diff
@@ -0,0 +1,27 @@
+// @ts-nocheck
+import { clusterApiUrl, Connection, Commitment, PublicKey } from '@solana/web3.js';
+import { TOKEN_PROGRAM } from './config/solana';
+
+export const createConnectionConfig = (clusterApi = clusterApiUrl('mainnet-beta'), commitment = 'confirmed') =>
+  new Connection(clusterApi, commitment as Commitment);
+
+export const getParsedAccountByMint = async ({ mintAddress, connection = createConnectionConfig() }) => {
+  try {
+    const res = await connection.getParsedProgramAccounts(new PublicKey(TOKEN_PROGRAM), {
+      filters: [
+        { dataSize: 165 },
+        {
+          memcmp: {
+            offset: 0,
+            bytes: mintAddress,
+          },
+        },
+      ],
+    });
+
+    return res[0];
+  } catch (err) {
+    console.error(err);
+    return null;
```

Comment:
I think we need to remove try-catch here and push the error further.
similar to what we did with 
https://github.com/NftEyez/sol-rayz/blob/main/packages/sol-rayz/src/getParsedNftAccountsByOwner.ts#L61

For now, looks like there is no way to identify errors outside of this function since it will catch all.

Reply:
okay